src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import defImage from "../../Assets/avatarDefault.png";
+import "./Avatar.scss";
+
+//this component expects a link to image source and not passed displays a default image
```

Comment:
Please, use multi-line commenting when documenting components. You can even specify the author if you want to.